raiden/tests/integration/transfer/test_mediatedtransfer.py

```diff
@@ -515,17 +475,7 @@ def run_test_mediated_transfer_with_allocated_fee(
     # any fee (the channel's fee was 0).
     # The second transfer's fee was deducted by
     # app1 (provided we've set the fee of the channel)
-    with gevent.Timeout(network_wait):
-        wait_assert(
-            assert_synced_channel_state,
-            token_network_address,
-            app1,
-            deposit - (amount * 2) - fee,
-            [],
-            app2,
-            deposit + (amount * 2) + fee,
-            [],
-        )
+    transfer_and_assert_path(raiden_network, token_address, amount, 2, fee, timeout)
```

Comment:
please, use the keyword arguments, otherwise one cannot tell what the `2` stands for